### PR TITLE
Pre launch changes

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: src/DocSidebar/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, {useState, useCallback} from 'react';
 import { WithBackgroundImage } from 'libra-docusaurus-components';
+import {OVERFLOW_CONTAINER_CLASS} from '@theme/Layout';
```
Comment (jjwill) [inline src/DocSidebar/index.js:3]: nit: spaces

File: src/Layout/index.js [hunk 2]
```diff
@@ -23,6 +23,25 @@ if (TEST_ADA) {
   axe(React, ReactDOM, 1000);
 }
 
+const scrollToTop = () =>
```
Comment (jjwill) [inline src/Layout/index.js:26]: Can you add this to a util class and pass in a selector?

File: src/Layout/index.js [hunk 3]
```diff
@@ -23,6 +23,25 @@ if (TEST_ADA) {
...
+  document.querySelector(`.${OVERFLOW_CONTAINER_CLASS}`).scrollTo(0, 0);
+
+const onLoad = () => {
+  const hash = window.location.hash;
+
+  if (!hash) {
+    scrollToTop();
+  } else {
+    setTimeout(() => {
```
Comment (jjwill) [inline src/Layout/index.js:35]: This looks like a hack. Why are we needing this?

Comment (oriooctopus) [reply src/Layout/index.js:35]: Yeah, it is. I looked into it for about 45 minutes to try and find a clean solution, and I'm sure one exists. Basically, by the time useEffect runs on Layout (I also tried on DocItem) the content hasn't fully loaded yet. Which means that if we scroll the y coordinate that we're really supposed to be at can be off, sometimes by as much as 1000 pixels at the bottom of a long page.

I would love to look into it further and find a clean solution, but I just didn't want to eat up a lot of time finding one.

Comment (jjwill) [reply src/Layout/index.js:35]: What's routing look like? They using react router? If so let's use a lib like https://github.com/rafgraph/react-router-hash-link/blob/master/src/index.js

Comment (oriooctopus) [reply src/Layout/index.js]: After further discussion we have decided to open an issue with docusaurus https://github.com/facebook/docusaurus/issues/3109

**Pull request merged**